### PR TITLE
Fix die of fate color

### PR DESCRIPTION
**Pull request by Readystorm546**
Die of fate color fix
:cl:  
bugfix: Fixed color for stealth die of fate 
/:cl:

### CODE REVIEW

**Comment by Hopekz** [comment]
Is this a meme PR?

File: code/modules/awaymissions/mission_code/Academy.dm [hunk 1]
```diff
@@ -165,6 +165,7 @@
 
 /obj/item/dice/d20/fate/stealth
 	name = "d20"
+	color = NULL
```
Comment (ToasterBiome) [inline code/modules/awaymissions/mission_code/Academy.dm]: if this isnt a meme pr then null should be lowercase iirc

File: code/modules/awaymissions/mission_code/Academy.dm [hunk 2]
```diff
@@ -182,6 +183,7 @@
 
 /obj/item/dice/d20/fate/cursed/stealth
 	name = "Die of Fate"
+	color = NULL
```
Comment (ToasterBiome) [inline code/modules/awaymissions/mission_code/Academy.dm]: same here

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
wow im gonna feel like a fool if this is a meme pr

**Comment by Readystorm546** [comment]
not a meme im just bad at code

**Comment by patpol4** [comment]
what

**Comment by JamieD1** [comment]
Why?

**Pull request merged**